aiosmtpd/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright 2014-2021 The aiosmtpd Developers
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.4.1"
+__version__ = "1.4.2a1"
```

Comment:
I originally wanted this to be `1.4.1-post1` because the change is minor.

However that goes against SemVer, which will consider as to be "pre"-release, so I'm bumping to 1.4.2 instead, even if the changes are so minor, current users of 1.4.1 totally does not need to upgrade.